src/pages/signup-affiliates/_types.ts

```diff
@@ -0,0 +1,85 @@
+import { Dispatch, ReactElement, SetStateAction } from 'react'
```

Comment:
Can we do some spacing format in this file, for better readability 🙏

Reply:
sure, have sense